userland/examples/nrf51_aes/aes.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <tock.h>
+
+#define DRIVER_ECB 34
+#define DRIVER_CCM 35
+#define KEY        0
+#define ENC        1
+#define DEC        2 
```

Comment:
#define AES_DRIVER 17
#define AES_KEY    0
#define AES_DATA   1 
#define AES_ENC    2
#define AES_DEC    3 
#define AES_CTR    4
ok? 

btw, changed signed char to unsigned char it considered as better practice for raw bytes.